lib/channel.js

```diff
@@ -26,6 +26,13 @@ function Channel(connection, name, options) {
     this.listening = null;
     this.name = name || 'mubsub';
 
+    this.recreate = options.recreate || true;
```

Comment:
this should probably be `this.recreate = options.recreate ?? true;` otherwise a false value will always be converted to true.

Reply:
Done